.github/workflows/pull-request-check.yaml

```
@@ -0,0 +1,40 @@
+name: Pull Request Check
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  lint:
+    name: Run lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local action
```

```suggestion
      - name: Checkout repository
```
`local action`が何を指し示しているか自分は理解できていませんが、[actions/checkoutのREADME](https://github.com/actions/checkout/blob/main/README.md)に
> This action checks-out your repository under $GITHUB_WORKSPACE, so your workflow can access it.

と記載があるので、`Checkout repository`などにしたいです